docs/source/posts/2023/2023-07-29-week-9-praneeth.rst

```diff
@@ -12,6 +12,10 @@ Continuing from the previous week, it seemed like we were almost done with the *
 
 The default behavior of *TextBox2D* now includes a dynamic bounding box, which scales automatically based on the contained text. Users can customize this option through a simple flag setting. However, this change affected some UI elements like *Combobox2d*, which relied on the default textbox size. Consequently, I had to make updates to ensure compatibility. Additionally, the default initialization of the *TextBlock2D* was completely static, which led to the possibility of the text extending beyond the background and failing certain tests. To tackle this, I made adjustments to the overflow helper function in the *test_elements.py* file. After a few tweaks and issue resolutions, the PR was ready for review and was successfully merged after passing the review process.
```

Comment:
*`Combobox2d`* -> *`ComboBox2D`*

Reply:
https://fury.gl/latest/reference/fury.ui.html#combobox2d

The UI mentions it this way only ie. `ComboBox2D` that why I mentioned it that way!

Reply:
Got it, its fine